HealthcareSystem/PatientWebApp/Controllers/PatientController.cs

```diff
@@ -156,6 +156,24 @@ public IActionResult ChangeImagePathForPatent(string jmbg, string name)
             }
             return RedirectPermanent("http://localhost:65117/html/patients_home_page.html");
         }
+        /// <summary>
+        /// /getting malicious patients(who canceled examinations 3 or more times in the past month)
+        /// </summary>
+        /// <returns>list of patients</returns>
+        [HttpGet("malicious-patients")]
+        public IActionResult GetMaliciousPatients()
```

Comment:
I maybe would've preferred if this path was just "malicious" so you'd end up with "patients/malicious" instead of "patients/malicious-patients".

Reply:
I agree